### ISSUE TITLE
Reproducing results on Deezer dataset

### ISSUE THREAD

**Issue by Chenhui1016** [qitianwu/NodeFormer#3]
Hi there,

Thanks for the fantastic work! 

I'm running `run.sh` and find out that the metric for Deezer is set to be "rocauc", while the paper uses accuracy as the metric on Deezer (shown in Figure 2). When I change the metric from "rocauc" to "acc" in `run.sh`, the averaged accuracy is 65.12%, which is much lower than the accuracy reported in the paper (~71%). Could you kindly let me know the proper hyperparameter setting for reproducing the results on Deezer? Thanks in advance!

**Comment by qitianwu** [comment]
Hi Chenhui,

The metric we used for Deezer is rocauc (for binary classification on this dataset) and the results can be reproduced using our provided hyper-parameters in run.sh. The "Accuracy" for Deezer in Figure 2 is a mistake and the plotted scores are indeed rocauc after double check. Sorry for causing the confusion and we will fix it in a new pdf version later uploaded on ResearchGate

**Comment by Chenhui1016** [comment]
Thanks for your prompt reply! I can reproduce NodeFormer's rocauc score on Deezer.

Btw, I just got the rocauc of 72.42 ± 0.38% on ogbn-proteins by using the hyperparameters in `run.sh`, which seems to be lower than 77.45 ± 1.15% reported in the paper. Could you kindly check those default hyperparameters on ogbn-proteins in `run.sh`?  Thanks again for your help!

**Comment by qitianwu** [comment]
I just checked for this, and we indeed use the hyper-parameters in run.sh for ogbn-proteins. We run the model on a RTX 2080Ti, and can reproduce the result. Can you tell me what GPU is used for your experiment? As shown by our experiments, the GPU types could have some influence on the results due to different generated random results

**Comment by Chenhui1016** [comment]
I'm using RTX A6000. Got it, thx!

**Comment by qitianwu** [comment]
Hi Chenhui,

Thank you again for checking our implementation. I have added the model checkpoints and code for testing on ogbn-proteins, which may help to make it more convenient to reproduce the score (77+) in our paper if needed for further research.
